use-cases/accounting.md

```diff
@@ -28,6 +28,14 @@ Before assisting any new client with connecting to your accounting integration,
 
 Ensure you follow our general [guidelines](../guidelines) for testing integrations.
 
+### Get and query rebated items
+
+You can use the API to find out if an accounting item is rebated. First use [Get all bills](../operations/finance.md#get-all-bills) to get a list of the order items on the bill. For each [order item](../operations/finance.md#order-item), examine the order item data to identify those items which are Rebates and if so, what is the [Rebated item ID](../operations/finance.md#rebate-order-item-data). You can then call [Get all accounting items](../operations/finance.md#get-all-accounting-items) with the Rebated item ID to return the details of the accounting item with that ID.
+
+Notes:
+1. You cannot use these operations to find out if an entire Bill has been rebated, only individual items in the Bill; all rebates are on an individual basis.
+2. You can find out what items have been rebated on a Bill by taking the Rebated item IDs from [Get all bills](../operations/finance.md#get-all-bills) and using them to call [Get all accounting items](../operations/finance.md#get-all-accounting-items).
```

Comment:
```suggestion
2. You can find out what items have been rebated on a Bill by taking the item IDs from [Get all bills](../operations/finance.md#get-all-bills) and using them as `RebatedOrderItemIds` filter when calling [Get all accounting items](../operations/finance.md#get-all-accounting-items). If there is an item with `RebatedOrderItemId` matching an `ID` of the original item, then that given original item has been rebated.
```

Reply:
It's a bit confusing - why are these notes here, when they're essentially the answer to the question in the first paragraph? We're not gonna have it duplicate here, right? Let's just consolidate it, so that things aren't here twice and it has some sense :)